internal/updater/install_test.go

```
@@ -0,0 +1,354 @@
+	t.Cleanup(srv.Close)
+	fix.server = srv
+
+	rel := Release{
+		TagName: "v" + latestVersion,
+		HTMLURL: srv.URL + "/release/v" + latestVersion,
+		Assets: []ReleaseAsset{
+			{Name: archiveName, BrowserDownloadURL: srv.URL + "/asset/" + archiveName},
+			{Name: "checksums.txt", BrowserDownloadURL: srv.URL + "/asset/checksums.txt"},
+		},
+	}
+	fix.releaseJSON, _ = json.Marshal(rel)
```

golangci-lint (2.11.4)

[error] Error return value of `encoding/json.Marshal` is not checked

(errchkjson)